sdcm/monitorstack/restore.py

```
@@ -393,13 +395,20 @@ def restore_sct_dashboards(grafana_docker_port, sct_dashboard_file):
         with open(annotations_file, encoding="utf-8") as f:
             annotations = json.load(f)
+    except json.JSONDecodeError as details:
+        LOGGER.warning(
+            "Annotations file %s contains invalid JSON: %s. Skip loading annotations", annotations_file, details
+        )
+        return False
 
+    try:
         annotations_url = f"http://localhost:{grafana_docker_port}/api/annotations"
         for an in annotations:
             res = requests.post(annotations_url, data=json.dumps(an), headers={"Content-Type": "application/json"})
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)